decidim-proposals/app/views/decidim/proposals/admin/proposal_answers/_form.html.erb

```diff
@@ -6,7 +6,7 @@
 
     <div class="card-section">
       <div class="row column">
-        <%= f.collection_radio_buttons :internal_state, [["accepted", t(".accepted")], ["rejected", t(".rejected")], ["evaluating", t(".evaluating")]], :first, :last, prompt: true %>
+        <%= f.collection_radio_buttons :internal_state, [["not_answered", t(".not_answered")], ["accepted", t(".accepted")], ["rejected", t(".rejected")], ["evaluating", t(".evaluating")]], :first, :last, prompt: true %>
```

Comment:
Shouldn't we also update the `answered_at` back to `nil` for this filter to work properly:
https://github.com/decidim/decidim/blob/a15c91cd006fec147d3c4705173e3ab0b9296923/decidim-proposals/app/models/decidim/proposals/proposal.rb#L59

Reply:
Yes, this was a doubt that I had when starting working with this issue, but didn't have a strong opinion regarding this. I think I even mentioned it on a conversation to @carolromero, but I don't remember if we reached to a conclusion

Reply:
Yep, we agreed to update it to `nil` in order to be coherent with the normal functioning of the filter.

Reply:
I've done this already. I have also added the spec checking for `answered_at` column and changed the other specs to check this column.